packages/model-viewer/src/features/scene-graph.ts

```diff
@@ -283,6 +295,33 @@ export const SceneGraphMixin = <T extends Constructor<ModelViewerElementBase>>(
     [$onModelGraftMutation](_event: Event) {
       this[$needsRender]();
     }
+
+    /** @export */
+    async exportScene(options: {
+			binary: false,
+			trs: false,
+			onlyVisible: true,
+			truncateDrawRange: true,
+			embedImages: true,
+			maxTextureSize: 1024,
```

Comment:
Why 1024?

Reply:
Just a sensible default value; it's taken directly from three's GLTFExporter

Reply:
Three defaults this to infinity, which I like better as a default since it's not destructive.